vector/src/main/java/im/vector/app/features/onboarding/ftueauth/FtueAuthSignUpSignInSelectionFragment.kt

```diff
@@ -60,27 +60,27 @@ class FtueAuthSignUpSignInSelectionFragment @Inject constructor() : AbstractSSOF
             ServerType.MatrixOrg -> {
```

Comment:
Extremely minor, maybe its worth creating a function like:
```
    private fun update(@DrawableRes drawableId: Int,
                       isServerIconVisible: Boolean,
                       signInTitle: String,
                       signInText: String
                       ){
        views.loginSignupSigninServerIcon.setImageResource(drawableId)
        views.loginSignupSigninServerIcon.isVisible = isServerIconVisible
        views.loginSignupSigninTitle.text = signInTitle
        views.loginSignupSigninText.text = signInText
    }
```
to reduce code duplication

Reply:
https://github.com/vector-im/element-android/pull/5691/commits/a2bcbfb7c730517a71cc1e1b3a183de646a37521

Reply:
Nice!